app/pathogen/arbovirus/dashboard/(map)/ArboCountryPopUpContent.tsx

```diff
@@ -1,6 +1,7 @@
 import { typedGroupBy } from "@/lib/utils"
 import { ShortformArbovirus, convertArboSFtoArbo } from "../(visualizations)/recharts"
 import { PopUpContentRow } from "./ArboStudyPopupContent";
+import { pathogenColors } from "./ArbovirusMap";
```

Comment:
Made many changes across the maps files: 

These mainly include the frosted glass finish, fixing the glitch bug and also ordering all the pathogens similarly across the application.

Reply:
Assigning that cursor glitch to you and moving it to in review :+1: